modules/Gallery/Gallery.tsx

```diff
@@ -1,7 +1,9 @@
 import type { NewsroomGallery } from '@prezly/sdk';
 import { getAssetsUrl, getUploadcareGroupUrl } from '@prezly/theme-kit-nextjs';
+import { useEffect, useState } from 'react';
 
 import { SlateRenderer } from '@/components';
+import StoryLinks from '@/components/StoryLinks';
```

Comment:
Let's add `StoryLinks` component to the index, so that we can import it from `@/components` as every other component in the theme.

Reply:
Updated